docs/benchmarks.md

```diff
@@ -1,209 +1,232 @@
 # GCSFuse Performance Benchmarks
 
-[FIO](https://fio.readthedocs.io/en/latest/) is used to perform load tests on
-GCSFuse. Below tables shows performance metrics of GCSFuse for different
+[fio](https://fio.readthedocs.io/en/latest/) is used to perform load tests on
+GCSFuse. The tables below show performance metrics of GCSFuse for different
 workloads for the given test setup:
 
 ## Test setup:
 
-* Infra: GCP VM
-* OS: ubuntu-20.04
-* Framework: FIO (version 3.39)
-* GCSFuse version: 2.11.1
+* Infra: GCP VM [C4-standard-192](https://cloud.google.com/compute/docs/general-purpose-machines#c4_series)
+* Network: [Tier_1](https://cloud.google.com/compute/docs/networking/configure-vm-with-high-bandwidth-configuration) Networking enabled on VM providing 200 Gbps egress bandwidth.
+* OS Version: [Ubuntu 22.04 LTS](https://cloud.google.com/compute/docs/images/os-details#notable-difference-ubuntu)
+* Image Family: [ubuntu-2204-lts](https://cloud.google.com/compute/docs/images/os-details#notable-difference-ubuntu)
+* Disk Type: [Hyperdisk Balanced](https://cloud.google.com/compute/docs/disks/hd-types/hyperdisk-balanced)
+* VM Region: us-south1
+* GCS Bucket ([HNS enabled](https://cloud.google.com/storage/docs/hns-overview)) Region: us-south1
+* Framework: fio (version 3.39)
+* GCSFuse version: [v3.4.3](https://github.com/GoogleCloudPlatform/gcsfuse/releases/tag/v3.4.3)
 
-## FIO workloads
-Please read the details about the FIO specification [here](https://fio.readthedocs.io/en/latest/).
-### Reads 
-  ```
+## Fio workloads
+Please read the details about the fio specification [here](https://fio.readthedocs.io/en/latest/).
```

Comment:
Can we remove this line as we already linked the fio documentation in the top line